test/e2e/gcp.go

```diff
@@ -0,0 +1,121 @@
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"google.golang.org/api/option"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strconv"
+	"time"
+
+	configv1 "github.com/openshift/api/config/v1"
+	dns "google.golang.org/api/dns/v1"
+)
+
+const (
+	defaultGCPProject = "openshift-gce-devel"
```

Comment:
Would be nice to get the project from the `Infrastructure` resource. It has platform specific data, like [projectID for GCP](https://pkg.go.dev/github.com/openshift/api/config/v1#GCPPlatformStatus) (`Infrastructure.Status.PlatformStatus.GCP.ProjectID`). Here is [an example](https://github.com/openshift/cluster-ingress-operator/blob/a1d6add3cf0ec66a68d64b6d050f261f13b87c90/pkg/operator/controller/ingress/controller.go#L215-L217) of how to get the infrastructure resource.

I think that this can be put in `operator_test.go` in the part specific to GCP. For non OpenShift deployment would be nice to read the project name from an environment variable, similar to the credentials.

Reply:
@alebedev87  done 👍🏽